### PR TITLE
OSDOCS#10143: Release note for no longer generating legacy SA token s…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…ecrets

Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10143

Link to docs preview:
https://76844--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-legacy-sa-tokens_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sanchezl Apologies for the delay, I know it's been awhile since we last talked about the proper wordings on this. Could you PTAL and let me know what you think on this release note for the changes in behavior?

Preview: https://76844--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-legacy-sa-tokens

**Comment by bergerhoffer** [comment]
@gangwgr can you PTAL for QE review as well? Thanks!

**Comment by gangwgr** [comment]
seems ok to me

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -303,6 +303,18 @@ With this release, the etcd tuning parameters can be set to values that optimize
 
 With this release, `dasd` image artifacts for the `s390x` architecture are removed from the {product-title} image building pipeline. You can still use the `metal4k` image artifact, which is identical and contains the same functionality.
 
+[discrete]
+[id="ocp-4-16-legacy-sa-tokens_{context}"]
+=== Legacy service account API token secrets are no longer generated for each service account
+
+Prior to {product-title} 4.16, an image pull secret and a service account API token secret were generated when a service account was created. Starting with {product-title} 4.16, the service account API token secret is no longer generated for each service account.
```
Comment (sanchezl) [inline release_notes/ocp-4-16-release-notes.adoc:310]:
```suggestion
Prior to {product-title} 4.16, when the integrated image registry is enabled, a legacy service account API token secret was generated for every service account in the cluster. Starting with {product-title} 4.16, the legacy service account API token secret is no longer generated for each service account.
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -303,6 +303,18 @@ With this release, the etcd tuning parameters can be set to values that optimize
...
+
+Additionally, the image pull secret no longer uses a legacy service account API token secret. Instead, the image pull secret now uses a bound service account token.
```
Comment (sanchezl) [inline release_notes/ocp-4-16-release-notes.adoc:312]:
```suggestion
Additionally,  when the integrated image registry is enabled, the image pull secret generated for every service account no longer uses a legacy service account API token. Instead, the image pull secret now uses a bound service account token that is automatically refreshed before it expires.
```

**Review by sanchezl — CHANGES_REQUESTED** [review]

**Comment by bergerhoffer** [comment]
@sanchezl Updated, PTAL

**Review by sanchezl — APPROVED** [review]
/lgtm

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

**Comment by mletalie** [comment]
Great job on this, @bergerhoffer!

**Pull request merged**